README.md

```diff
@@ -130,11 +130,7 @@ Debian 10        | yes
 FreeBSD 11.3     | yes
 FreeBSD 12.1     | yes
 Ubuntu 16.04     | ???
```

Comment:
zonemaster/zonemaster-engine#678 claims that 16.04 does not support algo 15. Should this row too be updated to reflect that?

Reply:
@mattias-p, you reviewed without my last patch yesterday. Please review again.

Reply:
Weird. I wonder how that happened.